server/queries/findAdminByCredentials.test.js

```diff
@@ -1,43 +1,42 @@
 /* eslint-disable no-undef */
-// const { ObjectID } = require('mongodb');
-// const { initDb, closeDb } = require('../databaseConnection');
+const { initDb, closeDb } = require('../databaseConnection');
 
-// const findAdminByCredentials = require('./findAdminByCredentials');
+const findAdminByCredentials = require('./findAdminByCredentials');
 
-it('Test to pass file', (done) => {
-  done();
+beforeEach(() => {
+  return initDb();
 });
 
-// beforeEach(() => {
-//   return initDb();
-// });
-
-// afterEach(() => {
-//   return closeDb();
-// });
+afterEach(() => {
+  return closeDb();
+});
 
-// it('Testing findAdminByCredentials returns one admin document when the correct credentials are provided.', async () => {
-//   expect.assertions(2);
-//   const adminDocument = await findAdminByCredentials('admin', 'admin');
-//   const id = ObjectID('519f1f77bcf86cd799439173');
-//   expect(adminDocument).toStrictEqual({
-//     _id: id,
-//     username: 'admin',
-//     password: 'admin',
-//     type: 'admin',
-//     employeeId: '123123',
-//   });
-//   expect(typeof adminDocument).toBe('object');
-// });
+it('Testing findAdminByCredentials returns one admin document when the correct credentials are provided.', async () => {
+  expect.assertions(6);
+  const adminDocument = await findAdminByCredentials('admin', 'admin');
+  expect(adminDocument.username).toStrictEqual('admin');
+  expect(adminDocument.password).toStrictEqual('admin');
+  expect(adminDocument.type).toStrictEqual('admin');
+  expect(adminDocument.employeeId).toStrictEqual('123123');
+  expect(typeof adminDocument).toBe('object');
+  expect(Object.keys(adminDocument)).toEqual([
+    '_id',
```

Comment:
dummyAdmins.js no longer has a `_id` property so won't this test fail?

Reply:
Yes it did, not sure what happened with dummyData!